pallets/messages/src/weights.rs

```diff
@@ -91,6 +91,7 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			// Standard Error: 139_000
 			.saturating_add(Weight::from_ref_time(4_711_000 as u64).saturating_mul(s as u64))
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
+			.saturating_add(T::DbWeight::get().writes(2 as u64))
```

Comment:
Where did these come from?

Also we should likely run benchmarks before merging right? Lots of weights could/should have changed

Reply:
Older weights than Main , need to run benchmarks before merge to update these